.claude/hooks/session-context.sh

```
@@ -0,0 +1,72 @@
+
+# Build a search query from branch context
+if [ -n "$TICKET" ]; then
+  # Try to get ticket title from GitHub (fast, cached)
+  TITLE=$(gh issue view "$TICKET" --json title --jq '.title' 2>/dev/null)
+  if [ -n "$TITLE" ]; then
+    QUERY="$TITLE"
+  else
+    QUERY="ticket $TICKET $(echo "$BRANCH" | tr '/-' ' ')"
+  fi
+else
+  QUERY=$(echo "$BRANCH" | tr '/-' ' ')
```

Shellcheck (0.11.0)

[warning] QUERY appears unused. Verify use (or export if used externally).

(SC2034)